appc.js

```diff
@@ -9,11 +9,13 @@ module.exports = {
     platforms: {
         ios: {
             vendor: {
+
+                // Q: How would we make sure this is unique accross modules?
                 'MyFramework': {
                      // these can be an array or string
-                     source: ['src'],
-                     header: 'src',
-                     resource: 'src'
+                     source: ['vendor'],
+                     header: 'vendor',
+                     resource: 'vendor'
```

Comment:
Agree on the naming convention, although it's not really a convention since it's configurable, but it makes sense both app and module template use the same. I'm not certain on the name though. I think `vendor` should only include stuff you don't check into versioning, so stuff pulled from registries. Maybe `src` was better afterall.